src/domain/context/aspect/aspect.service.ts

```diff
@@ -120,6 +120,10 @@ export class AspectService {
       aspect.description = aspectData.description;
     }
 
+    if (aspectData.typeDescription) {
```

Comment:
Not sure why we would need to update the typeDescription, it is fixed - and also questionable if it should be stored per entity - but lets not pick this up now.

Reply:
I agree that it's not ideal to be per entity, but that will open another set of updates, similarly to preference / preferenceType. I believe it should be updateable thou.